src/main/resources/experimental_materials.json

```diff
@@ -0,0 +1 @@
+["POTTED_PALE_OAK_SAPLING","PALE_OAK_WALL_SIGN","PALE_HANGING_MOSS","PALE_OAK_HANGING_SIGN","PALE_MOSS_BLOCK","PALE_OAK_SAPLING","STRIPPED_PALE_OAK_WOOD","PALE_OAK_LEAVES","PALE_MOSS_CARPET","PALE_OAK_WOOD","CREAKING_SPAWN_EGG","PALE_OAK_CHEST_BOAT","PALE_OAK_WALL_HANGING_SIGN","PALE_OAK_LOG","PALE_OAK_SIGN","PALE_OAK_FENCE","PALE_OAK_BOAT","STRIPPED_PALE_OAK_LOG","PALE_OAK_PRESSURE_PLATE","PALE_OAK_TRAPDOOR","PALE_OAK_DOOR","PALE_OAK_PLANKS","CREAKING_HEART","PALE_OAK_STAIRS","PALE_OAK_SLAB","PALE_OAK_BUTTON","PALE_OAK_FENCE_GATE"]
```

Comment:
Should this be `.gitignore`d?

Reply:
i think it's a good way of tracking what changes if we commit it.